tasks/config.yml

```diff
@@ -38,3 +35,17 @@
   lineinfile:
     dest: /etc/rancher/rke2/config.yaml
     line: "write-kubeconfig-mode: {{ rke2_write_kubeconfig_mode }}"
+
+# Airgap Images
+- name: Does the airgap images folder exist?
+  stat:
+      path: /var/lib/rancher/rke2/agent/images
+  register: rke2_agent_images
+
+- name: create the airgap images folder
+  ansible.builtin.file:
+    path: /var/lib/rancher/rke2/agent/images
+    state: directory
+    recurse: yes
+  when: not rke2_agent_images.stat.exists
+
```

Comment:
Forgive me if I'm misunderstanding your logic (it's late at night...), but tasks related to airgap installs should be optional, not default.  A user might be running these playbooks in a non-airgapped environment, in which case these tasks would not be relevant

Reply:
That is an excellent point. That section was there and I just made it idempotent.  I will create another PR with a variable for if the airgapped steps should take place or not.